common/include/obc_errors.h

```diff
@@ -0,0 +1,25 @@
+#ifndef COMMON_INCLUDE_OBC_ERRORS_H_
+#define COMMON_INCLUDE_OBC_ERRORS_H_
+
+typedef enum {
+    /* Common Errors 0 - 99 */
+    OBC_ERR_CODE_SUCCESS = 0,
+    OBC_ERR_CODE_UNKNOWN = 1,
+    OBC_ERR_CODE_INVALID_ARG = 2,
+    OBC_ERR_CODE_BUFF_TOO_SMALL = 3,
+
+    /* FreeRTOS Errors 100 - 199 */
+    OBC_ERR_CODE_QUEUE_FULL = 100,
+    OBC_ERR_CODE_MUTEX_TIMEOUT = 101,
+    OBC_ERR_CODE_TASK_CREATION_FAILED = 102,
+    OBC_ERR_CODE_TIMER_CREATION_FAILED = 103,
+    OBC_ERR_CODE_MUTEX_CREATION_FAILED = 104,
+
+    /* Driver Errors 200 - 299 */
+    OBC_ERR_CODE_GENERAL_SCI_FAILED = 200,
+    OBC_ERR_CODE_SCI_SEND_FAILED = 201,
+    OBC_ERR_CODE_SCI_RECEIVE_FAILED = 202,
+    OBC_ERR_CODE_SCI_INIT_NOT_CALLED = 203,
```

Comment:
similarly, could leave an error something for the bootcamper to add (its a small line change but it forces them to look at the code file and understand what they're doing a bit better)

Reply:
Yep that actually is part of the challenge lol

Reply:
LOL mb did not see that